nautobot/core/events/__init__.py

```diff
@@ -12,6 +16,43 @@
 logger = logging.getLogger(__name__)
 
 
+def load_event_brokers(nautobt_event_brokers):
+    """Process plugins and log errors if they can't be loaded."""
+    for publisher_name, publisher in nautobt_event_brokers.items():
```

Comment:
```suggestion
def load_event_brokers(event_broker_configs):
    """Process plugins and log errors if they can't be loaded."""
    for publisher_name, publisher in event_broker_configs.items():
```

Also, is there a reason you're using `publisher` verbiage throughout instead of `broker`?